### PR TITLE
Add package pcdsutils

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ZLLentz** [comment]
I will be a maintainer for this recipe

**Comment by hhslepicka** [comment]
I accept be a maintainer for this recipe

**Comment by hhslepicka** [comment]
@conda-forge/help-python this recipe is ready for review. CircleCI is failing at some unrelated point.

**Comment by klauer** [comment]
@conda-forge-admin please rerender

**Comment by klauer** [comment]
OK, all platforms are passing, only Circle is failing with what seems to be not the fault of pcdsutils/this PR: `/home/conda/staged-recipes/.circleci/build_steps.sh: line 48: CONFIG: unbound variable`

File: recipes/pcdsutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - pyqt >=5
+    - qtpy
```
Comment (scopatz) [inline recipes/pcdsutils/meta.yaml]: Are these really needed when pip runs?

Comment (klauer) [reply recipes/pcdsutils/meta.yaml]: They are not. Pushed a fix.

**Comment by hhslepicka** [comment]
@scopatz this is good for another review

File: recipes/pcdsutils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
```
Comment (scopatz) [inline recipes/pcdsutils/meta.yaml]: You don't need setuptools either. Please just leave pip and python :wink:

**Comment by klauer** [comment]
Force-pushed after a rebase on conda-forge/master, which apparently was causing the Circle CI failure.

File: recipes/pcdsutils/meta.yaml (hunk 2 above)
Comment (klauer) [reply recipes/pcdsutils/meta.yaml]: Thanks.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**